src/git-repository.coffee

```diff
@@ -101,8 +101,13 @@ class GitRepository
   # Public: Destroy this {GitRepository} object.
   #
   # This destroys any tasks and subscriptions and releases the underlying
-  # libgit2 repository handle.
+  # libgit2 repository handle. This method is idempotent.
   destroy: ->
+    if @emitter?
+      @emitter.emit 'did-destroy'
+      @emitter.off()
```

Comment:
Ah, it does appear to be from `event-kit`:

    {Emitter, Disposable, CompositeDisposable} = require 'event-kit'

Fixed.